website/pages/style.css

```diff
@@ -15,24 +15,22 @@
 @import '~@hashicorp/react-alert/dist/style.css';
 @import '~@hashicorp/react-button/styles/index.css';
 @import '~@hashicorp/react-call-to-action/dist/style.css';
-@import '~@hashicorp/react-code-block/dist/style.css';
-@import '~@hashicorp/react-consent-manager/dist/style.css';
```

Comment:
In cases where we're shipping styles un-transpiled, I've removed `/dist` from import paths. The updated `nextjs-scripts` has [a default setting to transpile `@hashicorp/react-*` modules](https://github.com/hashicorp/nextjs-scripts/blob/1e7f68a03f937e347c439ea0fd5c9cbfaf07c2eb/index.js#L28).